rpc/namespaces/eth/utils.go

```diff
@@ -0,0 +1,166 @@
+package eth
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/cosmos/ethermint/x/evm/types"
+
+	"github.com/ethereum/go-ethereum/core/vm"
+)
+
+const (
+	DefaultEVMErrorCode          = -32000
+	VMExecuteException           = -32015
+	VMExecuteExceptionInEstimate = 3
```

Comment:
where are the references to these status codes?

Reply:
to transform error to DataError

Reply:
i mean, where did -32015, -32000, and 3 get pulled from?

Reply:
`DefaultEVMErrorCode = -32000` was a default error code that pulled from ethereum
`VMExecuteExceptionInEstimate = 3` was pulled from remix during i debugging a contract
`VMExecuteException  = -32015` was pulled from truffle

Reply:
can we add those as comments in the code? that way we have a reference to these error codes? thanks!

Reply:
done